RecordCollection/Controllers/AlbumsController.cs

```diff
@@ -15,8 +19,39 @@ public AlbumsController(RecordCollectionContext context)
         public IActionResult Index()
         {
             var albums = _context.Albums.ToList();
-            
             return View(albums);
         }
+
+        [Route("Albums/{id:int}")]
+        public async Task<IActionResult> Show(int? id)
+        {
+            if (id == null)
```

Comment:
It is not required and the functionality of the code won't change, but you are able to use single-line If statements without the brackets. (ex. if (id == null) return NotFound();) It might help in cleaning up the code a bit and removing unnecessary brackets and whatnot.